week-3/E-array-map/exercise.js

```diff
@@ -2,4 +2,22 @@
 // Write multiple solutions using different syntax (as shown in the README)
 
 var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
+/*Case 1*/
 
+var multiplyNumbers = numbers.map(number => {
+  return number * 100;
+});
+
+console.log(multiplyNumbers);
+
+/*Case 2*/
+
+var multiplyNumbers = numbers.map(function(number) {
+  return number * 100;
+});
+
+/*Case 3 */
+
+var multiplyNumbers = numbers.map(function multiplyNumbers(number) {
+  return number * 100;
+});
```

Comment:
Nice work! They're all correct! There's 1 more potential that I can think of, can you think what it might be? Hint: you used it in your `.find()`  and `.every()` solutions